std_msgs/Header header

sq_grasping/grasp[] grasps

geometry_msgs/Vector3 table_center
